builds/misc/images-release.yaml

```diff
@@ -0,0 +1,602 @@
+          name: "Temperature Sensor"
+          project: SimulatedTemperatureSensor
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics
+        displayName: 'Build Image - azureiotedge-diagnostics - amd64'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch armv7l &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch armv7l
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM32'
+      - bash: |
+            scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(configuration) --target-arch aarch64 &&
+            scripts/linux/buildImage.sh -r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -i azureiotedge-diagnostics -n $(namespace) -P azureiotedge-diagnostics --target-arch aarch64
+        displayName: 'Build Image - azureiotedge-diagnostics - ARM64'
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifacts to VSTS'
+        inputs:
+          PathtoPublish: '$(Build.BinariesDirectory)/publish'
+          ArtifactName: 'publish-linux'
+################################################################################
+  - job: windows
+################################################################################
+    displayName: Windows
+    pool:
+      name: Azure-IoT-Edge-Core
+      demands: 
+        - Build-Image -equals true
+        - win-rs5
+        - DotNetFramework
+    variables:
+      build.registry.address: edgebuilds.azurecr.io
+      build.registry.user: EdgeBuilds
+      registry.address: edgerelease.azurecr.io
+      registry.user: EdgeRelease
+    steps: 
+      - script: |
+            docker login $(build.registry.address) -u $(build.registry.user) -p $(build.registry.password)
+            docker login $(registry.address) -u $(registry.user) -p $(registry.password)
+        displayName: 'docker login'
+      - template: ../templates/install-dotnet3.yaml
+      - powershell: "scripts/windows/build/Publish-Branch.ps1 -Configuration:\"$(configuration)\" -PublishTests:$False -UpdateVersion"
+        displayName: "Build ($(configuration))"
+        name: build
+      - task: PowerShell@2
+        displayName: "Build iotedge-diagnostics"
+        inputs:
+          filePath: ./edgelet/build/windows/build-diagnostics.ps1
+          targetType: filePath
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
```

Comment:
why don't need  - template: ../templates/dotnet2-globaljson.yaml here?

Reply:
windows doesn't need this logic